events/api.py

```diff
@@ -558,12 +558,14 @@ def validate_data_source(self, value):
     def validate_publisher(self, value):
         if value:
             # TODO: allow optionally declaring publisher in any user admin org (regular org for drafts)
-            if value not in (self.publisher, self.publisher.replaced_by):
+            # TODO: self.user.admin_organizations.all() or user.get_admin_organizations_and_descendants()?
+            if value not in (self.publisher, self.publisher.replaced_by) and\
+                    value not in self.user.admin_organizations.all():
```

Comment:
Actually, here `self.publisher` is one of the orgs in `self.user.admin_organizations.all()`.

However, we must also check `replaced_by` just in case, so perhaps better just use `if value not in (self.user.admin_organizations.all() | map(lambda x: getattr(x, 'replaced_by'))` or something similar, to check all admin_organizations and their replacements?